src/clients/api/queries/useUserMarketInfo.ts

```diff
@@ -58,71 +63,130 @@ const useUserMarketInfo = ({
+      }
+
+      let walletBalance = new BigNumber(0);
+      let supplyBalance = new BigNumber(0);
+      let borrowBalance = new BigNumber(0);
+      let isEnabled = false;
+      const percentOfLimit = '0';
+
+      const wallet = balances[vtokenAddress];
+      if (accountAddress && wallet) {
+        walletBalance = toDecimalAmount(wallet.tokenBalance);
+        supplyBalance = toDecimalAmount(wallet.balanceOfUnderlying);
+        borrowBalance = toDecimalAmount(wallet.borrowBalanceCurrent);
+        if (item.id === 'bnb') {
+          isEnabled = true;
+        } else {
+          isEnabled = toDecimalAmount(wallet.tokenAllowance).isGreaterThan(walletBalance);
+        }
+      }
+
+      const asset = {
+        key: index,
+        id: item.id,
+        img: item.asset,
+        vimg: item.vasset,
+        symbol: market.underlyingSymbol || item.id.toUpperCase(),
+        decimals: item.decimals,
+        tokenAddress: market.underlyingAddress,
+        vsymbol: market.symbol,
+        vtokenAddress,
+        supplyApy: new BigNumber(market.supplyApy || 0),
+        borrowApy: new BigNumber(market.borrowApy || 0),
+        xvsSupplyApy: new BigNumber(market.supplyVenusApy || 0),
+        xvsBorrowApy: new BigNumber(market.borrowVenusApy || 0),
+        collateralFactor: new BigNumber(market.collateralFactor || 0).div(1e18),
+        tokenPrice: new BigNumber(market.tokenPrice || 0),
+        liquidity: new BigNumber(market.liquidity || 0),
+        borrowCaps: new BigNumber(market.borrowCaps || 0),
+        treasuryTotalBorrowsUsd: new BigNumber(market.totalBorrowsUsd),
+        treasuryTotalSupplyUsd: new BigNumber(market.totalSupplyUsd),
+        treasuryTotalSupply: new BigNumber(market.totalSupply),
+        treasuryTotalBorrows: new BigNumber(market.totalBorrows2),
+        treasuryBalance,
+        walletBalance,
+        supplyBalance,
+        borrowBalance,
+        isEnabled,
+        collateral,
+        percentOfLimit,
+        hypotheticalLiquidity: ['0', '0', '0'] as [string, string, string],
+      };
+      // user totals
+      const borrowBalanceUSD = asset.borrowBalance.times(asset.tokenPrice);
+      const supplyBalanceUSD = asset.supplyBalance.times(asset.tokenPrice);
+      acc.userTotalBorrowBalance = acc.userTotalBorrowBalance.plus(borrowBalanceUSD);
+      acc.userTotalSupplyBalance = acc.userTotalSupplyBalance.plus(supplyBalanceUSD);
+
+      // treasury totals
+      acc.treasuryTotalUsdBalance = acc.treasuryTotalUsdBalance.plus(
+        asset.treasuryBalance.multipliedBy(asset.tokenPrice),
+      );
+      acc.treasuryTotalSupplyUsdBalance = acc.treasuryTotalSupplyUsdBalance.plus(
+        asset.treasuryTotalSupplyUsd,
+      );
+      acc.treasuryTotalBorrowUsdBalance = acc.treasuryTotalBorrowUsdBalance.plus(
+        asset.treasuryTotalBorrowsUsd,
+      );
+      acc.treasuryTotalAvailableLiquidityUsdBalance =
+        acc.treasuryTotalAvailableLiquidityUsdBalance.plus(asset.liquidity);
```

Comment:
Since we're refactoring that I think the USD values should be expressed in cents to be consistent with the rest of the codebase:
<img width="736" alt="Screen Shot 2022-05-11 at 08 58 29" src="https://user-images.githubusercontent.com/44675210/167798916-700fc620-e4b2-4c40-a2b1-9cd86fc32484.png">